controller/engine_controller.go

```diff
@@ -1668,13 +1667,12 @@ func (ec *EngineController) UpgradeEngineProcess(e *longhorn.Engine) error {
 	}
 	defer c.Close()
 
-	sizeRequested, err := ec.ds.IsEngineImageCLIAPIVersionEqualToOrLargerThan(e.Spec.EngineImage, 6)
+	engineCLIAPIVersion, err := ec.ds.GetEngineImageCLIAPIVersion(e.Spec.EngineImage)
```

Comment:
`IsEngineImageCLIAPIVersionEqualToOrLargerThan` seems not used anymore, so suggest removing.

Reply:
Removed. Thank you.